### PR TITLE
add controller, dto

### PR DESCRIPTION
**Pull request by Alkey**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -52,7 +60,39 @@
...
+            <groupId>javax.servlet.jsp.jstl</groupId>
+            <artifactId>javax.servlet.jsp.jstl-api</artifactId>
+            <version>1.2.1</version>
+        </dependency>
+
+        <dependency>
+            <groupId>taglibs</groupId>
+            <artifactId>standard</artifactId>
+            <version>1.1.2</version>
+        </dependency>
```
Comment (matviichuk92) [inline pom.xml]: I think we should delete dependencies for jsp and jstl

File: src/main/java/spring/config/WebConfig.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+import org.springframework.web.servlet.view.InternalResourceViewResolver;
+import org.springframework.web.servlet.view.JstlView;
+
+@Configuration
+@EnableWebMvc
+@ComponentScan(basePackages = {"spring.controllers"})
+public class WebConfig {
+
+    @Bean
+    public InternalResourceViewResolver resolver() {
```
Comment (matviichuk92) [inline src/main/java/spring/config/WebConfig.java]: I think we should delete this bean too

**Review by matviichuk92 — COMMENTED** [review]
good job!

File: src/main/java/spring/config/AppConfig.java [hunk 3]
```diff
@@ -19,7 +19,7 @@
         "spring.dao"
 })
 public class AppConfig {
-    private Environment environment;
+    private final Environment environment;
```
Comment (RomanDubovskyi) [inline src/main/java/spring/config/AppConfig.java:22]: why do you use "final" here, but don't use it with UserService and UserDao, let's keep your code consistent

**Review by RomanDubovskyi — APPROVED** [review]

**Pull request merged**